start.sh

```diff
@@ -1 +1 @@
-bundle && bundle exec jekyll serve --trace --open-url --livereload --future
+hugo serve --disableFastRender --buildFuture --buildDrafts --buildExpired
```

Comment:
You mentioned some issues with the live reload feature, which I'm guessing is why the `disableFastRender` flag is passed to local server.

If logic lives in a shortcode, that can sometimes be the culprit. See [this issue](https://github.com/gohugoio/hugo/issues/6177) on Hugo's repository. Usually, this isn't a big deal, but it may explain some of your frustrations.

For example, in the file `content/lessons/_index.md` you call a short code to render the lesson gallery. If I had to guess, this may of been where you experienced live reload issues. Moving this logic to a template such as `list.html` should fix this and the `disableFastRender` flag can be removed for faster rendering.

If you were experiencing live reload issues somewhere else, let me know and I'm happy to take a look!

Reply:
I can't recall exactly where I ran into these problems, it seemed pretty random. One thing I do remember is that `<title>` in `<head>` wasn't being updated often. I'm certainly going to keep the flag there for my own uses to make sure I never run into that problem.